Description: Timestamps in fpc documentation are hindering reproducible builds
 Don't add the date to the footer of docs generated by fpdoc
Author: Paul Gevers <elbrus@debian.org>
Forwarded: no
Index: fpc/fpcdocs/Makefile.fpc
===================================================================
--- fpc.orig/fpcdocs/Makefile.fpc
+++ fpc/fpcdocs/Makefile.fpc
@@ -115,7 +115,7 @@ ifeq ($(HIDEPROTECTED),YES)
 FCLOPTS+= --hide-protected
 endif
 
-FPDOCHTMLOPTS=--footer-date="mmm dd yyyy"
+FPDOCHTMLOPTS=
 
 ifeq (chm,$(HTMLFMT))
   HTMLSUFFIX:=.chm
